projects/js-packages/publicize-components/src/components/post-publish-review-prompt/index.js

```diff
@@ -0,0 +1,57 @@
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { getJetpackData } from '@automattic/jetpack-shared-extension-utils';
+import apiFetch from '@wordpress/api-fetch';
+import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { useCallback, useState } from '@wordpress/element';
+import usePublicizeConfig from '../../hooks/use-publicize-config';
+import { usePostStartedPublishing } from '../../hooks/use-saving-post';
+import useSocialMediaConnections from '../../hooks/use-social-media-connections';
+import ReviewPrompt from '../review-prompt';
+
+const PostPublishReviewPropmpt = () => {
+	const [ isReviewRequestDismissed, setIsReviewRequestDismissed ] = useState(
+		getJetpackData()?.social?.reviewRequestDismissed ?? true
```

Comment:
Since we do not have `reviewRequestDismissed` in the Jetpack plugin, this will always end up returning `true`. Is this okay?
https://github.com/Automattic/jetpack/blob/540064cf9bf3ac2618147e26b71c2c61f83bb3c2/projects/plugins/jetpack/class.jetpack-gutenberg.php#L726

Reply:
Ah I missed this comment somehow. Some weird GH UI stuff.

That's right, but I've ended up changing it back and then conditionally using the Social or Jetpack script name based on the active plugin.